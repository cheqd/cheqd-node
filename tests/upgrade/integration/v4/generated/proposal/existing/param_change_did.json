{
    "messages": [
        {
            "@type": "/cheqd.did.v2.MsgUpdateParams",
            "authority": "cheqd10d07y265gmmuvt4z0w9aw880jnsr700j5ql9az",
            "params": {
                "create_did": {
                    "denom": "ncheq",
                    "amount": "100000000000"
                },
                "update_did": {
                    "denom": "ncheq",
                    "amount": "50000000000"
                },
                "deactivate_did": {
                    "denom": "ncheq",
                    "amount": "20000000000"
                },
                "burn_factor": "0.990000000000000000"
            }
        }
    ],
    "metadata": "",
    "deposit": "10000000000ncheq",
    "title": "Did Param Change",
    "summary": "This is a simple example of a param change - double the values, 99% burn factor",
    "expedited": false
}