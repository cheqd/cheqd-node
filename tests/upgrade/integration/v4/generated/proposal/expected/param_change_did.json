{
  "create_did": [
    {
      "denom": "ncheq",
      "min_amount": "100000000000",
      "max_amount": "100000000000"
    }
  ],
  "update_did": [
    {
      "denom": "ncheq",
      "min_amount": "50000000000",
      "max_amount": "50000000000"
    }
  ],
  "deactivate_did": [
    {
      "denom": "ncheq",
      "min_amount": "20000000000",
      "max_amount": "20000000000"
    }
  ],
  "burn_factor": "0.990000000000000000"
}
